README.md

```diff
@@ -42,7 +42,7 @@ Clone the repository, change to the directory containing the repository.
 $ pip install -r requirements.txt
```

Comment:
Thanks @kytrinyx for adding the link.

The section on installing Python, in particular, is very good since it is well maintained and up to date since the workshops run pretty much every weekend. https://tutorial.djangogirls.org/en/python_installation/